src/main/scala/viper/silver/reporter/package.scala

```diff
@@ -16,7 +18,56 @@ package object reporter {
 
   type Position = viper.silver.ast.SourcePosition
 
+
+  trait SymbolType {
```

Comment:
What's a "SymbolType"? Just "Symbol" seems clearer, and more generally applicable, to me.

Reply:
I agree that the term "type" is not quite right here. But instances of this class represent not symbols themselves but "kinds" of symbols, indicating whether this declaration is for a function, a field, a variable, etc. 

I'm not aware of any standard term that we use in Viper for this purpose. We have the term "member" which means somethings else. 

Would ```SymbolKind``` sound okay to you?

Reply:
Yes, let's go with that. `SymbolKind` sounds less suggestive.